core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -789,6 +789,8 @@ SharedAllocationRecord<Kokkos::CudaHostPinnedSpace, void>
 // Iterate records to print orphaned memory ...
 void SharedAllocationRecord<Kokkos::CudaSpace, void>::print_records(
     std::ostream &s, const Kokkos::CudaSpace &, bool detail) {
+  (void)s;
+  (void)detail;
```

Comment:
from experience we know that this doesn't work on all compilers. but we can try.

Reply:
How about `std::ignore`?

Reply:
> from experience we know that this doesn't work on all compilers. but we can try.

Do you have any example we should look into?

> How about `std::ignore`?

That won't work in device code so we can't use it everywhere as a replacement.

Reply:
This function right here is not marked `__host__ __device__`

Reply:
I know. If possible, I would go with a common style. That's why I wrote "everywhere".